src/components/SiteItem.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+const site = props => (
```

Comment:
The name of the stateless react component should mirror that of the file name! `SiteItem`